CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
    * ADDED: PBF output for matrix endpoint [#4121](https://github.com/valhalla/valhalla/pull/4121)
    * CHANGED: sped up the transit gtfs ingestion process by sorting the feeds before querying them and avoiding copying their structures. forked just_gtfs into the valhalla org to accomplish it [#4167](https://github.com/valhalla/valhalla/pull/4167)
    * CHANGED: write traffic tile headers in `valhalla_build_extract` [#4195](https://github.com/valhalla/valhalla/pull/4195)
+   * ADDED: `source_percent_along` & `trace_attribute` to /trace_attribtues JSON response [#4199](https://github.com/valhalla/valhalla/pull/4199)
```

Comment:
ok it is pretty cool that the spell check shows up in the diff, i think for all the complaining i did about that pr this part is worth it. however i think also you wanted: 
```suggestion
   * ADDED: `source_percent_along` & `target_percent_along` to /trace_attributes JSON response [#4199](https://github.com/valhalla/valhalla/pull/4199)
```

Reply:
thanks, done